config/vufind/facets.ini

```diff
@@ -73,7 +73,9 @@ dateRange[] = publishDate
 ;       for a large number of building facets where top level is organization and
 ;       second level the library branch)
 ; all = Sort all levels alphabetically
-;hierarchicalFacetSortOptions[building] = top
```

Comment:
Is there a specific reason why the building example was removed? (Not that I'm hugely concerned about this, but I see that the comments above describe this example, so it might be worth keeping it here).

Reply:
No reason, reverted this.

Reply:
Thanks!